<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="author" content="Carles Hernandez-Ferrer ">
<meta name="description" content="The following is a detailed list of software I created or which I contributed in any way. The order follows my contribution time to them and not their release time.
Released Treatabolome DB
Short description: Although next-generation sequencing drastically improved diagnosis for patients with rare diseases, access to knowledge of effective treatments is still sparse and often unclear. Treatabolome DB is a database of rare disease-specific treatments mapped at gene and variant level to speed upt the identification of existing therapies at the time of clinical diagnosis." />
<meta name="keywords" content="bioinformatics, software, data analysis" />
<meta name="robots" content="noodp" />
<meta name="theme-color" content="" />
<link rel="canonical" href="http://212.71.254.23/software/" />


    <title>
        
            Software :: Carles Hernandez-Ferrer 
        
    </title>



<link href="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.5.0/css/flag-icon.min.css" rel="stylesheet"
    type="text/css">



<link rel="stylesheet" href="/main.0dbe19bf135ce9f0272c4f7c4b805c55427a2a421a96241af7daa8532e45dea9.css">






<meta itemprop="name" content="Software">
<meta itemprop="description" content="The following is a detailed list of software I created or which I contributed in any way. The order follows my contribution time to them and not their release time.
Released Treatabolome DB
Short description: Although next-generation sequencing drastically improved diagnosis for patients with rare diseases, access to knowledge of effective treatments is still sparse and often unclear. Treatabolome DB is a database of rare disease-specific treatments mapped at gene and variant level to speed upt the identification of existing therapies at the time of clinical diagnosis."><meta itemprop="datePublished" content="2022-01-23T16:46:07+01:00" />
<meta itemprop="dateModified" content="2022-01-23T16:46:07+01:00" />
<meta itemprop="wordCount" content="727"><meta itemprop="image" content="http://212.71.254.23"/>
<meta itemprop="keywords" content="" />
<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:image" content="http://212.71.254.23"/>

<meta name="twitter:title" content="Software"/>
<meta name="twitter:description" content="The following is a detailed list of software I created or which I contributed in any way. The order follows my contribution time to them and not their release time.
Released Treatabolome DB
Short description: Although next-generation sequencing drastically improved diagnosis for patients with rare diseases, access to knowledge of effective treatments is still sparse and often unclear. Treatabolome DB is a database of rare disease-specific treatments mapped at gene and variant level to speed upt the identification of existing therapies at the time of clinical diagnosis."/>








    <meta property="article:published_time" content="2022-01-23 16:46:07 &#43;0100 CET" />








    </head>

    
        <body>
    
    
        <div class="container">
            <header class="header">
    <span class="header__inner">
        <a href="/" style="text-decoration: none;">
    <div class="logo">
        
            <span class="logo__mark">></span>
            <span class="logo__text">C:\carles.exe</span>
            <span class="logo__cursor" style=
                  "
                   background-color:#4169E1;
                   ">
            </span>
        
    </div>
</a>


        <span class="header__right">
            
                <nav class="menu">
    <ul class="menu__inner"><li><a href="/about">About</a></li><li><a href="/academic">Academic</a></li><li><a href="/blog">Blog</a></li><li><a href="/software">Software</a></li>
    </ul>
</nav>

                <span class="menu-trigger">
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
                        <path d="M0 0h24v24H0z" fill="none"/>
                        <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/>
                    </svg>
                </span>
            
        </span>
    </span>
</header>


            <div class="content">
                
    <main class="post">

        <div class="post-info">
            
            </p>
        </div>

        <article>
            <h2 class="post-title"><a href="http://212.71.254.23/software/">Software</a></h2>

            
            
            

            <div class="post-content">
                <p>The following is a detailed list of software I created or which I contributed in any way. The order follows my contribution time to them and not their release time.</p>
<h4 id="released">Released</h4>
<p><strong><code>Treatabolome DB</code></strong></p>
<ul>
<li><strong>Short description</strong>: Although next-generation sequencing drastically improved diagnosis for patients with rare diseases, access to knowledge of effective treatments is still sparse and often unclear. Treatabolome DB is a database of rare disease-specific treatments mapped at gene and variant level to speed upt the identification of existing therapies at the time of clinical diagnosis.</li>
<li><strong>Role</strong>: Author (Transferred to <a href="https://cnag.es/teams/bioinformatics-unit/data-analysis">S Beltran and A Corvo</a>)</li>
<li><strong>Type</strong>: Web application (<a href="https://treatabolome.org/">access</a>)</li>
<li><strong>Publication(s)</strong>:
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8203244/">G Bonne, J Neuromuscul Dis., 2021</a></li>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/33682723/">A Atalaya; J Neuromuscul Dis.; 2021</a></li>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/30931400/">R Thompson;  Emerg Top Life Sci.; 2019</a></li>
</ul>
</li>
</ul>
<p><strong>RD-Connect Genome-Phenome Analysis Platform</strong></p>
<ul>
<li><strong>Short description</strong>: The &ldquo;RD-Connect GPAP&rdquo; was a result of the European Union FP7 funded RD-Connect project. It&rsquo;s goal is to accelerate rare disease research by connecting databases, patient registries, biobanks and clinical data through an intuitive user-friendly platform.</li>
<li><strong>Role</strong>: Maintainer/Contributor</li>
<li><strong>Type</strong>: Web application (<a href="https://platform.rd-connect.eu/">access</a>, <a href="https://bio.tools/rd-connect_platform">bio.tools</a>)</li>
<li><strong>Publication</strong>: <a href="https://www.nature.com/articles/s41431-021-00852-7">L Matalonga; Eur. J. Hum. Genet.; 2021</a></li>
</ul>
<p><strong><code>WEScover</code></strong></p>
<ul>
<li><strong>Short description</strong>: While whole exome sequencing is widely adopted in clinical and research settings, the potential false negatives due to incomplete breadth and depth of coverage for several exons in clinically implicated genes is a major concern. <code>WEScover</code> is a tool to quickly identify if all genes of interest would be reliably covered by WES or a targeted gene panel testing should be prioritized.</li>
<li><strong>Role</strong>: Contributor</li>
<li><strong>Type</strong>: Web application (<a href="https://tom.tch.harvard.edu/shinyapps/WEScover/">access</a>, <a href="https://bio.tools/wescover">bio.tools</a>)</li>
<li><strong>Publication</strong>: <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04178-5">IH Lee; BMC Bioinform.; 2021</a></li>
</ul>
<p><strong>Clinical Genome and Ancestry Report</strong></p>
<ul>
<li><strong>Short description</strong>: The &ldquo;CGAR&rdquo; was developed to enables users with no knowledge on genomic annotation to identify and prioritize phenotype-associated variants from exome and genome sequencing data through a user-friendly online platform.</li>
<li><strong>Role</strong>: Contributor</li>
<li><strong>Type</strong>: Web application (<a href="https://tom.tch.harvard.edu/apps/cgar/">access</a>, <a href="https://bio.tools/CGAR">bio.tools</a>)</li>
<li><strong>Publication</strong>: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.23942">IH Lee; Hum. Mutat.; 2019</a></li>
</ul>
<p><strong><code>rexposome</code> and <code>omicRexposome</code></strong></p>
<ul>
<li><strong>Short description</strong>: The study of the high-dimensional exposures experienced across the life course (aka. exposome) offers a new paradigm for investigating environmental contributions to disease etiology. The R packages <code>rexposome</code> and <code>omicRexposome</code> filled the lack of tools for managing, visualizing and analyzing the exposome and its association with health outcomes and integration with molecular signatures.</li>
<li><strong>Role</strong>: Author (Transferred to <a href="https://www.isglobal.org/en/our-team/-/profiles/18510">JR Gonzalez</a>)</li>
<li><strong>Type</strong>: R package
<ul>
<li><code>rexposome</code> (<a href="https://www.bioconductor.org/packages/release/bioc/html/rexposome.html">Bioconductor</a>, <a href="https://bio.tools/rexposome">bio.tools</a>)</li>
<li><code>omicRexposome</code> (<a href="https://www.bioconductor.org/packages/release/bioc/html/omicRexposome.html">Bioconductor</a>, <a href="https://bio.tools/omicrexposome">bio.tools</a>)</li>
</ul>
</li>
<li><strong>Publication(s)</strong>:
<ul>
<li><a href="https://academic.oup.com/bioinformatics/article/35/24/5344/5523848">C Hernandez-Ferrer; Bioinformatics; 2019</a></li>
<li><a href="https://www.nature.com/articles/s41467-022-34422-2">L Maitre; Nat.Com.; 2020</a></li>
</ul>
</li>
</ul>
<p><strong><code>psygenet2r</code></strong></p>
<ul>
<li><strong>Short description</strong>: While psychiatric disorders have a great impact on morbidity and genotypeâ€“phenotype resources for psychiatric diseases are key to enable the translation of research findings to a better care of patients, the <code>psygenet2r</code> packages allows to integrate the data from PsyGeNET (a knowledge resource on psychiatric diseases and their genes) in downstream network-medicine analysis of psychiatric disorders.</li>
<li><strong>Role</strong>: Author/Contributor</li>
<li><strong>Type</strong>: R package (<a href="https://www.bioconductor.org/packages/release/bioc/html/psygenet2r.html">Bioconductor</a>, <a href="https://bio.tools/psygenet2r">bio.tools</a>)</li>
<li><strong>Publication</strong>: <a href="https://academic.oup.com/bioinformatics/article/33/24/4004/4083576">A Gutierrez-Sacristan; Bioinformatics; 2017</a></li>
</ul>
<p><strong><code>MultiDataSet</code></strong></p>
<ul>
<li><strong>Short description</strong>: <code>MultiDataSet</code> was released to supply the need for a data-structure in R to manage multiple layers of information for the same set of individuals. Latter on, it was extended as a resource to manage large amounts of multiple and non-complete biomedical-related data while offering a simple and general way of sub-setting features and selecting samples.</li>
<li><strong>Role</strong>: Author (Transferred to <a href="https://www.isglobal.org/en/our-team/-/profiles/18510">JR Gonzalez</a>)</li>
<li><strong>Type</strong>: R package (<a href="https://www.bioconductor.org/packages/release/bioc/html/MultiDataSet.html">Bioconductor</a>, <a href="https://bio.tools/multidataset">bio.tools</a>)</li>
<li><strong>Publication</strong>: <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1455-1">C Hernandez-Ferrer; BMC Bioinform.; 2017</a></li>
</ul>
<h4 id="in-development">In development</h4>
<p><strong><code>epimutacions</code></strong></p>
<ul>
<li><strong>Short description</strong>: <code>epimutacions</code> (the name of the package is in Catalan) is a package that uses microarray DNA methylation data to identify epimutations, defined as a sequence of CpGs with aberrant methylation level but not enough to be discovered in case-control studies. The goal is to implement and collect already published methods, compute the burden of epimutations in a cohort of generally healthy population, and apply the method to accelerate rare disease diagnostic.</li>
<li><strong>Role</strong>: Contributor</li>
<li><strong>Type</strong>: R package (<a href="https://bioconductor.org/packages/release/bioc/html/epimutacions.html">Bioconductor</a>, <a href="https://bio.tools/epimutacions">bio.tools</a>)</li>
<li><strong>Publication</strong>: <em>in development</em></li>
</ul>
<h4 id="deprecated-or-discontinued">Deprecated or discontinued</h4>
<p><strong><code>CTDquerier</code></strong></p>
<ul>
<li><strong>Short description</strong>: <code>CTDquerier</code> was build to easily integrate data from the <a href="http://ctdbase.org/">Comparative Toxicogenomics Database</a> in biomedical studies that included genomic and/or environmental factors for studying the etiology of human diseases.</li>
<li><strong>Role</strong>: Author (Transferred to <a href="https://www.isglobal.org/en/our-team/-/profiles/18510">JR Gonzalez</a>)</li>
<li><strong>Type</strong>: R package (<a href="https://www.bioconductor.org/packages/release/bioc/html/CTDquerier.html">Bioconductor</a>, <a href="https://bio.tools/ctdquerier">bio.tools</a>)</li>
<li><strong>Publication</strong>: <a href="https://academic.oup.com/bioinformatics/article/34/18/3235/4983065">C Hernandez-Ferrer; Bioinformatics; 2018</a></li>
</ul>
<p><strong><code>affy2sv</code></strong></p>
<ul>
<li><strong>Short description</strong>: The R package <code>affy2sv</code> was build in order to incorporate data from the most common Affymetrix SNP array to downstream R pipelines identifying different genomic aberrations like chromosomal inversion or copy number variants.</li>
<li><strong>Role</strong>: Author (Transferred to <a href="https://www.isglobal.org/en/our-team/-/profiles/18510">JR Gonzalez</a>)</li>
<li><strong>Type</strong>: R package - see it&rsquo;s source code in <a href="https://github.com/isglobal-brge/affy2sv">github</a></li>
<li><strong>Publication</strong>: <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0608-y">C Hernandez-Ferrer; BMC Bioinform.; 2015</a></li>
</ul>

            </div>
        </article>

        <hr />

        <div class="post-info">
            
            
  		</div>
    </main>

            </div>

            
                <footer class="footer">
    <div class="footer__inner">
        <div class="footer__content">
            <span>&copy; 2023</span>
            <span><a href="http://212.71.254.23">Carles Hernandez-Ferrer</a></span>
            <span><a href="https://creativecommons.org/licenses/by-nc/4.0/" target="_blank" rel="noopener">CC BY-NC 4.0</a></span><span><a href="http://212.71.254.23/blog/index.xml" target="_blank" title="rss"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 20 20" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-rss"><path d="M4 11a9 9 0 0 1 9 9"></path><path d="M4 4a16 16 0 0 1 16 16"></path><circle cx="5" cy="19" r="1"></circle></svg></a></span>
        </div>
    </div>
    <div class="footer__inner">
        <div class="footer__content">
            <span>Powered by <a href="http://gohugo.io">Hugo</a></span>
            <span>Theme by <a href="https://github.com/rhazdon">Djordje Atlialp</a></span>
          </div>
    </div>
</footer>

            
        </div>

        



<script type="text/javascript" src="/bundle.min.bb2c6bc3ed452ca4759660e4020811f248bc2320081559e8a32d8b0092773852941133639d35e8370d03d3ddaa750b1edd6b343c5bd22a55d5bdeae8f648f49b.js" integrity="sha512-uyxrw&#43;1FLKR1lmDkAggR8ki8IyAIFVnooy2LAJJ3OFKUETNjnTXoNw0D092qdQse3Ws0PFvSKlXVvero9kj0mw=="></script>



    </body>
</html>
